From 7b09d79de3734a79896995c6b71e2005618bfeac Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Wed, 10 Jan 2018 23:36:43 +0000
Subject: [PATCH] joyent/manta#17 Typo - MUSKIE_MULTI_DC Reviewed by: Brian
 Bennett <brian.bennett@joyent.com> Reviewed by: Tim Kordas
 <tim.kordas@joyent.com> Approved by: Brian Bennett <brian.bennett@joyent.com>

---
 docs/operator-guide.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/operator-guide.md b/docs/operator-guide.md
index 8372784..323d7e4 100644
--- a/docs/operator-guide.md
+++ b/docs/operator-guide.md
@@ -847,7 +847,7 @@ multi-DC, multi-compute-node deployment.  The general process is:
     This is required to enforce that object writes are distributed to multiple
     datacenters.  In the SAPI master datacenter:
 
-        headnode $ app_uuid="$(sdc-sapi /applications?name=manta | json -H uuid)"
+        headnode $ app_uuid="$(sdc-sapi /applications?name=manta | json -Ha uuid)"
         headnode $ echo '{ "metadata": { "MUSKIE_MULTI_DC": true } }' | \
             sapiadm update "$app_uuid"
 
-- 
2.21.0

